Doc/library/importlib.metadata.rst

```diff
@@ -301,6 +358,15 @@ function::
 Mapping import to distribution packages
 ---------------------------------------
 
+.. function:: packages_distributions()
```

Comment:
I find this function name a bit confusing.

Reply:
Indeed, but it's an existing API.